### DISCUSSION TITLE
Setting Up OpenRadioss Calculations + ParaView on AWS EC2

### DISCUSSION THREAD

**Discussion by shanevit** [OpenRadioss/OpenRadioss#3180] [Q&A]
Hello everyone,

I’m planning to set up and run calculations using OpenRadioss on an AWS EC2 cloud. Additionally, I aim to install ParaView for post-processing the results. I’m relatively new to this setup (cloud computing) and was wondering if anyone here has experience with such a workflow or could share some tips and best practices.

Specifically, I’d like to know:

- What’s the best way to configure an EC2 instance for running OpenRadioss? Any recommendations?
- Are there any guides or pitfalls to be aware of during the installation of OpenRadioss on AWS?
- Tips for installing and using ParaView for result visualization, especially if there are any issues with remote rendering or handling large datasets on AWS.

Any advice, links to relevant documentation, or personal experiences would be greatly appreciated!

Thank you in advance!
Vitek

**Comment by OlivierWienholtz** [comment]
Hello,

for OpenRadioss there are not any pitfails.
I would also have a look at compatibility with Paraview & how virtual Graphic cards are supported. 

I believe a Linux instance will be easier to handle than a Windows instance.
It'll be depending what type of instances you choose and if you plan to run on Single Machine or create a Cluster.
Later will be more complex to handle.

On the instances : I took a brief look at the current available instances. 
The "G" series M8G, M7G, M6G are ARM processors. You can build OpenRadioss for that Platform (arch=linuxa64)
the i-series : M7i, M7i-flex are Intel Based Intances. OpenRadioss releases will work.
The a-series : M6a, M5a are AMD based instances. OpenRadioss releases will work.

Hope it helps,
Olivier
